doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -23,7 +23,7 @@ This action initiates a series of questions about the dispute:
 
 After Joe answers the questions, the web site provides Joe with an incident number, `000004`.
 
-Fortress Bank must now decide whether to reimburse the disputed amount without investigation or whether to investigate the claim. Investigation requires more resources so automatic acceptance of the disputed amount costs the bank less in terms of resources. However, if the bank automatically accepts all disputed amounts the cost will ultimately be greater to the bank because of money paid for fraudulent claims. Someone or something must make the decision to investigate or not.
+Fortress Bank must now decide whether to reimburse the disputed amount without human investigation or whether to investigate the claim manually. Manual investigation requires more resources so automatic processing of the disputed amount costs the bank less in terms of human resources. However, if the bank automatically accepts all disputed amounts the cost will ultimately be greater to the bank because of money paid for fraudulent claims. Someone or something must make the decision to investigate or not.
```

Comment:
I suggest a minor rewrite for "Someone or something must make the decision to investigate or not."

Someone or something must make the decision whether or not to investigate.

Reply:
Good!